README.md

```diff
@@ -1,6 +1,6 @@
-# Project Name
+# Frontdoor terraform script with App Service
```

Comment:
nit: The official name is "Azure Front Door" might want to separate Frontdoor into two words.

Reply:
Updated to Bhargav's input from below